web_tool_script_3.R

```diff
@@ -41,6 +41,7 @@ scenario_geography <- "Global"
 audit_file <- read_csv(file.path(proc_input_path, portfolio_name, "audit_file.csv"))
 emissions <- read_rds(file.path(proc_input_path, portfolio_name, "emissions.rda"))
 
+# load equity portfolio data
 if (file.exists(file.path(results_path, portfolio_name, "Equity_results_portfolio.rda"))) {
```

Comment:
Nice! Nicer would be `if (has_low_equity_portolio_data())` ...

Reply:
I would add that as something to refactor since I am not sure of the optimal level of abstraction here atm

Reply:
https://github.com/2DegreesInvesting/PACTA_analysis/issues/213